### PR TITLE
Add Crypto to DB

### PR DESCRIPTION
**Pull request by cnohall**
This PR is small and only has the purpose of adding crypto to the `blockonomics_bitcoin_orders` table.

### CODE REVIEW

File: upgrade/install-1.7.92.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    $object = $object;
+    $install = $install;
+    updateDatabase();
+    return true; //if there were no errors
+}
+
+//function used to upgrade the module table
+function updateDatabase()
+{
+    $query = 'ALTER TABLE `'._DB_PREFIX_.'blockonomics_bitcoin_orders'.'` ADD `crypto` varchar(191) NOT NULL';
```
Comment (DarrenWestwood) [inline upgrade/install-1.7.92.php:36]: The upgrade process is working well to add the column during the upgrade. varchar(3) can instead be used to keep the column in line with the WordPress plugin.

Comment (cnohall) [reply upgrade/install-1.7.92.php]: Done

**Review by DarrenWestwood — APPROVED** [review]

**Review by DarrenWestwood — CHANGES_REQUESTED** [review]
The changes work for upgrading users, however, on new installations, the crypto column is missing:
https://github.com/blockonomics/prestashop-plugin/blob/1.7/blockonomics.php#L161

**Comment by cnohall** [comment]
Good catch! I've added crypto to installDB().

**Review by DarrenWestwood — APPROVED** [review]

File: upgrade/install-1.7.92.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    updateDatabase();
+    return true; //if there were no errors
+}
+
+//function used to upgrade the module table
+function updateDatabase()
+{
+    $query = 'ALTER TABLE `'._DB_PREFIX_.'blockonomics_bitcoin_orders'.'` ADD `crypto` varchar(3) NOT NULL';
+    if (!Db::getInstance()->execute($query)) {
+        return false;
```
Comment (shivaenigma) [inline upgrade/install-1.7.92.php:38]: Return value is never used in caller, also it is not returning true on success.

**Review by shivaenigma — CHANGES_REQUESTED** [review]

File: upgrade/install-1.7.92.php (hunk 2 above)
Comment (cnohall) [reply upgrade/install-1.7.92.php:37]: Now it `updateDatabase()` returns true on success. `upgrade_module_1_7_92()` now returns the result of `updateDatabase()`, false if any errors, true if successful.

**Pull request merged**